staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -1127,6 +1126,6 @@ func recordDecodeError(groupResource schema.GroupResource, key string) {
 }
 
 // getTypeName returns type name of an object for reporting purposes.
-func getTypeName(obj interface{}) string {
```

Comment:
Just a little hint from gopls

Reply:
Alright, I'll remove it. Just thought it would be a good housekeeping alongside this PR. Sorry for that 👍

Reply:
Housekeeping is good, but we would have enabled gopls and enforced it in the CI if we cared about this specific linter. Mixing cleanup into non-cleanup related PR obfuscates and make review longer.